student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,10 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     const char* expected = my_username();
+     char* actual = malloc_username_from_conf_file();
+     
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(expected, actual,"OK");
```

Comment:
@Jahnavi-pinnamaneni the purpose of message here was to indicate mismatch in usernames.